rpc/grpc/client_server.go

```diff
@@ -61,7 +62,13 @@ func StartGRPCServer(env *core.Environment, ln net.Listener) error {
 //
 // Deprecated: A new gRPC API will be introduced after v0.38.
 func StartGRPCClient(protoAddr string) BroadcastAPIClient {
-	conn, err := grpc.Dial(protoAddr, grpc.WithTransportCredentials(insecure.NewCredentials()), grpc.WithContextDialer(dialerFunc))
+	parsedAddr := protoAddr
+
+	// Remove "tcp://" and "unix://" prefix if it exists
```

Comment:
grpc address parsing supports unix and dns, but doesn't support tcp or other prefixes. Please update this to follow that. You could use a regex